vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/model/VGSDate.kt

```diff
@@ -0,0 +1,72 @@
+package com.verygoodsecurity.vgscollect.core.model
+
+import java.util.*
+
+public class VGSDate private constructor(
```

Comment:
@DonaldRG `public` - visibility modifier can be removed. Also, I would make this `class` an `data class`.